Person/src/test/edu.stanford/LibraryPatronTest.java

```diff
@@ -0,0 +1,73 @@
+package edu.stanford;
+
+import org.junit.Test;
+import javax.xml.transform.Source;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
+import java.io.*;
+
+import static org.junit.Assert.assertEquals;
+
+public class LibraryPatronTest {
+
+    @Test
+    public void hasActiveIDs() throws Exception {
+        TransformerFactory factory = TransformerFactory.newInstance();
```

Comment:
If we ever need another test here that uses this, it could be abstracted to a method that returns a Transformer based on a given file.

Reply:
Fixed in https://github.com/sul-dlss/registry-harvester/pull/90/commits/5b3b6f8dd86de3f340360b6ab2c97af37a473236